syntheseus/cli/eval_single_step.py

```diff
@@ -432,7 +442,10 @@ def run_from_config(
     if OmegaConf.is_missing(config.back_translation_config, "model_class"):
         back_translation_model = None
     else:
-        back_translation_model = get_model_fn(config.back_translation_config)
+        # Cast is ok because back translation code calls "is_forward"
```

Comment:
Suggesting a change as initially I couldn't parse what this comment is telling me:
```suggestion
        # Back translation model must be a forward model (checked in `compute_metrics`).
```